console/console-init/ui/src/components/JsonEditor/JsonEditor.tsx

```diff
@@ -34,7 +36,11 @@ const JsonEditor: React.FunctionComponent<IJsonEditorProps> = ({
   const [value, setValue] = useState<string | undefined>(jsonValue);
 
   const onChange = (value: string) => {
-    setValue(value);
+    if (manageState) {
```

Comment:
This condition is not really make sense. We need to avoid to passing same data again  in editor.
Keep the previous code and just add one more useEffect as given below and try with it.

 useEffect(()=>{
    setValue(jsonValue);
  },[jsonValue]);